ethergo/submitter/submitter.go

```
@@ -388,13 +388,13 @@
 	// this also prevents a bug in the caller from breaking our lock
 	transactor.Nonce = new(big.Int).Add(new(big.Int).SetUint64(math.MaxUint64), big.NewInt(1))
 
+	//tmpdebug
```

golangci-lint (1.62.2)

commentFormatting: put a space between `//` and comment text

(gocritic)